packages/flutter/lib/src/material/button_theme.dart

```diff
@@ -284,6 +330,382 @@ class ButtonThemeData extends Diagnosticable {
   /// This property only affects [DropdownButton] and its menu.
   final bool alignedDropdown;
 
+  /// The background fill color for [RaisedButton]s.
+  ///
+  /// This property is null by default.
+  ///
+  /// See also:
+  ///
+  ///  * [getFillColor], which is used by [RaisedButton] to compute its
+  ///    background fill color.
+  final Color buttonColor;
+
+  /// The background fill color for disabled [RaisedButton]s.
+  ///
+  /// Null by default.
+  ///
+  /// See also:
+  ///
+  ///  * [getDisabledFillColor], which is used by [RaisedButton] to compute its
+  ///    background fill color.
+  final Color disabledColor;
+
+  /// The color of the overlay that appears when a button is pressed.
+  ///
+  /// Null by default.
```

Comment:
Nit: here and else where it should be a lowercase `null`-the-value instead of uppcase `Null`-the-type

Reply:
OK, done.